BaseFilter/sections/specific.txt

```diff
@@ -283,6 +283,7 @@ chortle.co.uk,coloring.ws,dltk-holidays.com,dltk-kids.com,kidzone.ws,pcsteps.com
 !
 !
 coindesk.com##div[class^="high-impact-vertstyles__StyledWrapper-"]
+||h5.roulax.io/intgame/vue/roulaxAd/roulaxAd.js
```

Comment:
Please add `||roulax.io^$third-party` to adservers instead.

Reply:
OK, I will add